### ISSUE TITLE
Broken with new version of SyncLounge

### ISSUE THREAD

**Issue by brettg0396** [Starbix/docker-synclounge#14]
Since the apparent update of app.synclounge.tv a few days ago (UI overhaul, not sure what under the hood changes accompany it), the server no longer appears to be working. Attempting to join yields no result (clicking join after entering room credentials doesn't do anything) and the container logs say only the following (grep'd to exclude lines containing "Connected users", and with some info redacted):

```
Setting up with serverRoot of /slserver
SyncLounge Server successfully started on port ####
Someone connected to the ptserver socket
Invalid join attempt {
  roomId: 'movie',
  password: 'night',                                                desiredUsername: '############',
  desiredPartyPausingEnabled: true,
  desiredAutoHostEnabled: false,
  thumb: '#############',
  syncFlexibility: 3000,
  time: 0,
  duration: 0,
  playbackRate: 0,
  state: 'stopped',
  media: null,
  playerProduct: 'SyncLounge'
}

> SyncLounge@2.0.0 server /opt/synclounge
> node server.js

Setting up with serverRoot of /slserver
SyncLounge Server successfully started on port ####
```

Not sure if this is something on my end, but I can confirm that this container worked literally the day before the changes to app.synclounge.tv, so I am inclined to believe that something has changed in the way connections are made.

**Comment by ttshivers** [comment]
Yes. Some breaking changes were made between v2 and v3 of SyncLounge (hence the major version increase). It sounds like you are wanting to use `app.synclounge.tv` for the web app but have it connect to your own socket server (which is what I assume you are running this container for). Is that right?

If you just want to host the socket server, use the `synclounge/syncloungeserver` docker images.
https://hub.docker.com/r/synclounge/syncloungeserver

If you are also wanting to host both the web app and socket server, use the `synclounge/synclounge` docker images. https://hub.docker.com/r/synclounge/synclounge.

We abide by the semantic versioning guarantees, so in the future, you shouldn't find any breaking features unless there is a major version change (like there was from v2 to v3).

We are also still in the process of updating the documentation for the new version, so while that is happening, don't be afraid to ask any questions about configuring, etc.

**Comment by brettg0396** [comment]
Yes, that's the idea. Should I just run both then? Everything's behind an nginx reverse proxy so I don't mind hosting the web app as well if that will fix it.

**Comment by ttshivers** [comment]
Sure! It sounds like you want to completely self host, and the new version makes it very easy to do that. 
https://github.com/synclounge/synclounge#docker

Sample nginx configs:
https://github.com/synclounge/synclounge#sample-nginx-config
